frontend/components/main.jsx

```diff
@@ -62,34 +90,30 @@ export default class Main extends React.Component {
   }
 
   render() {
-    const firstRow = [
-      { name: PEOPLE['tim'].displayName, to: PEOPLE['tim'].path, key: 1, src: PEOPLE['tim'].src },
-      { name: PEOPLE['jay'].displayName, to: PEOPLE['jay'].path, key: 2, src: PEOPLE['jay'].src },
-      { name: PEOPLE['ryan'].displayName, to: PEOPLE['ryan'].path, key: 3, src: PEOPLE['ryan'].src },
-    ];
+    const special = { name: PEOPLE['matt'].displayName, to: PEOPLE['matt'].path, key: 4, src: PEOPLE['matt'].src };
 
-    const secondRow = [
-      { name: PEOPLE['matt'].displayName, to: PEOPLE['matt'].path, key: 4, src: PEOPLE['matt'].src },
+    const everyone = [
       { name: PEOPLE['kristi'].displayName, to: PEOPLE['kristi'].path, key: 5, src: PEOPLE['kristi'].src },
-    ];
-
-    const thirdRow = [
+      { name: PEOPLE['tim'].displayName, to: PEOPLE['tim'].path, key: 1, src: PEOPLE['tim'].src },
       { name: PEOPLE['erica'].displayName, to: PEOPLE['erica'].path, key: 6, src: PEOPLE['erica'].src },
+      { name: PEOPLE['jay'].displayName, to: PEOPLE['jay'].path, key: 2, src: PEOPLE['jay'].src },
       { name: PEOPLE['maryann'].displayName, to: PEOPLE['maryann'].path, key: 7, src: PEOPLE['maryann'].src },
+      { name: PEOPLE['ryan'].displayName, to: PEOPLE['ryan'].path, key: 3, src: PEOPLE['ryan'].src },
       { name: PEOPLE['helena'].displayName, to: PEOPLE['helena'].path, key: 8, src: PEOPLE['helena'].src },
     ];
 
     return (
       <div>
         <WelcomeHeader {...this.props} />
         <Table>
-          {firstRow.map(this.renderPerson)}
-        </Table>
-        <Table>
-          {secondRow.map(this.renderPerson)}
-        </Table>
-        <Table>
-          {thirdRow.map(this.renderPerson)}
+          <StyleLink to={special.to} key={special.key}>
+            <MattContainer>
+              <Person src={special.src} />
+              <Name>{special.name}</Name>
+              <Favorite><Star /></Favorite>
```

Comment:
might be able to avoid special casing this by adding a `special` property in the people objects that signifies whether to render `<Favorite...` in `this.renderPerson`